src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,24 @@
+package duke.command;
+
+import duke.main.Storage;
+import duke.main.TaskList;
+import duke.main.Ui;
+
+/**
+ * Abstract parent class for all commands.
+ */
+public abstract class Command {
+    /**
+     * Behaviour of command when run.
+     * @param taskList TaskList object of the Duke object for the Command object to use.
+     * @param ui Ui object of the Duke object for the Command object to use.
+     * @param storage Storage object of the Duke object for the Command object to use.
+     */
```

Comment:
Take note to leave an empty line between description and parameter section or all Javadoc comments.